### PR TITLE
Make the rewards pull from the Utopian API; fallback to current data.

### PR DESCRIPTION
**Pull request by netuoso**
Added the ability for the rewards to automatically pull data from https://utopian.io/api/stats. If the website requesting the data is not specified in Utopians CORS policy, the data will fallback to the current stats.

Function set on a 5 second interval to keep data constant for users.

Replaced the idea bulbs in the "ideas" section with font-awesome graphics set to 4x size.

Removed Octocat and replaced with Utopian logo. Altered the Github related text to Utopian content.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -96,7 +96,7 @@ <h2 class="text-white text-center">Earn rewards by contributing to your<br>favor
                         <h3 class="font-weight-bold my-4">Hi! I'm Octocat!</h3>
                         I work at GitHub, as a Mascot! It's a very great place to be. We are a world spanning community of enthusiasts and volunteers, working together on thousands of cool projects. <b>Utopian.io</b> rewards you for making contributions to Open Source projects, like fixing bugs, translating texts, writing documentation or sharing good ideas and more.<br>
                         <div class="text-right mt-3">
-                            <a href="#" class="btn btn-primary">Join our community!</a>
+                            <a href="https://v2.steemconnect.com/oauth2/authorize?client_id=utopian.app&redirect_uri=https%3A%2F%2Futopian.io%2Fcallback&scope=" class="btn btn-primary">Join our community!</a>
```
Comment (netuoso) [inline index.html]: This change makes the "Join or Community" button direct the user to a signin page.

File: index.html [hunk 2]
```diff
@@ -46,10 +46,10 @@
                     </li>
                 </ul>
                 <div class="text-right">
-                    <a href="#" class="btn btn-primary mr-3 font-weight-bold" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <a href="https://steemit.com/pick_account" class="btn btn-primary mr-3 font-weight-bold">
```
Comment (netuoso) [inline index.html]: Removed the navigation toggle from these links and made them active.

**Comment by netuoso** [comment]
@mktcode I pulled in the latest master and removed the conflicts. If you don't want the navigation links for "sign up" and "login" changed then just cherry picky the PR without those commits before merging.

**Comment by mktcode** [comment]
I think it's fine now. Thank you! :)

**Pull request merged**